Assets/Project/Modules/PlayerAnchor/Scripts/AbilityUnlock/Unlocker/ChainedOrb/AbilityUnlockerChainedOrbView.cs

```diff
@@ -53,25 +62,36 @@ private void OnDestroy()
 
             _orbHitPS = Instantiate(typeViewData.OrbHitParticlesPrefab, _orbHitParticlesHolder);
             _pickAbilityPS = Instantiate(typeViewData.PickAbilityParticlesPrefab, _pickAbilityParticlesHolder);
+
+            _particleFactory = ServiceLocator.Instance.GetService<IParticleFactory>();
         }
 
 
         public async UniTaskVoid PlayIdleAnimation()
         {
             _movingChainsSoundId = _audio.StartPlayingMovingChainsSound(gameObject);
+
+            _orbitalChains0.DOBlendableLocalRotateBy(Vector3.up, 180).SetLoops(-1);
+            _orbitalChains0.DOBlendableLocalRotateBy(Vector3.up, 60).SetLoops(-1);
+            _orbitalChains0.DOBlendableLocalRotateBy(Vector3.up, 120).SetLoops(-1);
         }
         
         public async UniTask PlayUnlockAbilityAnimation()
         {
             _audio.PlayHitSound(gameObject);
-            await _chainedOrb.PlayDisappearAnimation(_orbTargetTransform.position);
+            Transform sparks = _particleFactory.Create(_viewConfig.OnHitSparklesParticleType, Vector3.zero, quaternion.identity, transform);
+            sparks.LookAt(_orbTargetTransform.position);
+            sparks.transform.position += Vector3.forward * 0.3f;
+            sparks.transform.position += Vector3.up * 0.1f;
```

Comment:
The same with this displacement vector, it should be a Vector3 parameter in the `ChainedOrbViewConfig`  (Vector3(0, 0.1f, 0.3f) )

Reply:
Or just have another Transform field with the position to spawn the sparks